projects/js-packages/ai-client/src/components/ai-control/index.tsx

```diff
@@ -60,6 +61,29 @@ export default function AIControl( {
 } ) {
 	const promptUserInputRef = useRef( null );
 
+	useKeyboardShortcut(
+		[ 'command+enter', 'ctrl+enter' ],
```

Comment:
```suggestion
		'mod+enter',
```
https://craig.is/killing/mice